acceptance/openstack/ims/CreateImageFromECS_test.go

```diff
@@ -0,0 +1,68 @@
+package ims
+
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/openstack"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/tools"
+	image1 "github.com/opentelekomcloud/gophertelekomcloud/openstack/ims/v1/images"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/ims/v1/others"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/ims/v2/images"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/obs"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestCreateImageFromECS(t *testing.T) {
+	client1, client2 := getClient(t)
+
+	computeClient, err := clients.NewComputeV1Client()
+	th.AssertNoErr(t, err)
+
+	ecs := openstack.CreateCloudServer(t, computeClient, openstack.GetCloudServerCreateOpts(t))
+	t.Cleanup(func() { openstack.DeleteCloudServer(t, computeClient, ecs.ID) })
+
+	fromDisk, err := images.CreateImageFromDisk(client2, images.CreateImageFromDiskOpts{
+		Name:      tools.RandomString("ims-test-", 3),
+		VolumeId:  ecs.VolumeAttached[0].ID,
+		OsVersion: "Debian GNU/Linux 10.0.0 64bit",
+	})
+	th.AssertNoErr(t, err)
+	jobEntities(t, client1, client2, fromDisk)
+
+	fromECS, err := images.CreateImageFromECS(client2, images.CreateImageFromECSOpts{
```

Comment:
Also need to add: https://docs.otc.t-systems.com/image-management-service/api-ref/ims_apis/image/updating_image_information.html
and test for it

Reply:
`openstack/ims/v2/images/UpdateImage.go`
already here.